src/Components/LyricsModal.js

```diff
@@ -1,68 +1,103 @@
 import React, { useEffect, useState } from "react";
 import "./LyricsModal.css";
-import $ from "jquery";
 import "bootstrap";
 import "popper.js";
 import { getLyrics } from "../axios";
 import logo from "./../assets/Dog-swimming.png";
+import BeatLoader from "react-spinners/BeatLoader";
+import Modal from "react-modal";
 
-function Modal({ children, artist }) {
+function OpenModal({
```

Comment:
This should just be called `Modal` or `LyricsModal` or something as opposed to `OpenModal`.

Reply:
yes, ok